diff --git a/node_modules/@material-ui/core/ButtonBase/ButtonBase.js b/node_modules/@material-ui/core/ButtonBase/ButtonBase.js
index 4972129..36ce725 100644
--- a/node_modules/@material-ui/core/ButtonBase/ButtonBase.js
+++ b/node_modules/@material-ui/core/ButtonBase/ButtonBase.js
@@ -159,7 +159,7 @@ var ButtonBase = /*#__PURE__*/React.forwardRef(function ButtonBase(props, ref) {
     };
   }, []);
   React.useEffect(function () {
-    if (focusVisible && focusRipple && !disableRipple) {
+    if (focusVisible && focusRipple && !disableRipple && rippleRef.current) {
       rippleRef.current.pulsate();
     }
   }, [disableRipple, focusRipple, focusVisible]);
diff --git a/node_modules/@material-ui/core/esm/ButtonBase/ButtonBase.js b/node_modules/@material-ui/core/esm/ButtonBase/ButtonBase.js
index 58cfc64..92786c3 100644
--- a/node_modules/@material-ui/core/esm/ButtonBase/ButtonBase.js
+++ b/node_modules/@material-ui/core/esm/ButtonBase/ButtonBase.js
@@ -135,7 +135,7 @@ var ButtonBase = /*#__PURE__*/React.forwardRef(function ButtonBase(props, ref) {
     };
   }, []);
   React.useEffect(function () {
-    if (focusVisible && focusRipple && !disableRipple) {
+    if (focusVisible && focusRipple && !disableRipple && rippleRef.current) {
       rippleRef.current.pulsate();
     }
   }, [disableRipple, focusRipple, focusVisible]);
