diff --git a/node_modules/react-scripts/config/webpack.config.js b/node_modules/react-scripts/config/webpack.config.js
index 26c2a65..ca5ccd8 100644
--- a/node_modules/react-scripts/config/webpack.config.js
+++ b/node_modules/react-scripts/config/webpack.config.js
@@ -403,6 +403,8 @@ module.exports = function (webpackEnv) {
             // The preset includes JSX, Flow, TypeScript, and some ESnext features.
             {
               test: /\.(js|mjs|jsx|ts|tsx)$/,
+              // Exclude locale files that are huge don't need babel processing:
+              exclude: path.join(paths.appSrc, 'locale'),
               include: paths.appSrc,
               loader: require.resolve('babel-loader'),
               options: {
@@ -466,7 +468,10 @@ module.exports = function (webpackEnv) {
             // Unlike the application JS, we only compile the standard ES features.
             {
               test: /\.(js|mjs)$/,
-              exclude: /@babel(?:\/|\\{1,2})runtime/,
+              exclude: [/@babel(?:\/|\\{1,2})runtime/, 
+                // Exclude locale files that are huge don't need babel processing:
+                path.join(paths.appSrc, 'locale')
+              ],
               loader: require.resolve('babel-loader'),
               options: {
                 babelrc: false,
